### PR TITLE
tests/kola/basic: check for unlabeled files in /var and /etc

### PR DESCRIPTION
**Pull request by jlebon**
Inspired by https://github.com/coreos/fedora-coreos-config/pull/485.

### CODE REVIEW

**Comment by jlebon** [comment]
This actually uncovered an unlabeled directory: https://github.com/coreos/ignition/pull/1006. So it'll probably break CI until we respin Ignition with that fix.

**Comment by jlebon** [comment]
(That said, there doesn't seem to be any repercussions from that directory being unlabeled. Which makes sense, otherwise we would've noticed this much faster.)

**Review by dustymabe — APPROVED** [review]
LGTM

File: tests/kola/basic [hunk 1]
```diff
@@ -1,2 +1,14 @@
 #!/bin/bash
-exec systemctl is-enabled logrotate.service
+set -xeuo pipefail
+
+# check that logrotate is enabled
+systemctl is-enabled logrotate.service
+
+# check that no files are unlabeled
+for dir in /var /etc; do
+    find ${dir} -context '*:unlabeled_t:*' |& tee out.txt
```
Comment (bgilbert) [inline tests/kola/basic]: Can't we just `find /var /etc -context ...`?

Comment (jlebon) [reply tests/kola/basic]: Thanks, that's clearly better! :)

**Comment by bgilbert** [comment]
Test failure:

```
tee: out.txt: Operation not permitted
```

**Review by bgilbert — APPROVED** [review]

**Comment by cgwalters** [comment]
I took the opportunity to rebase :surfing_man: this, LGTM

**Pull request merged**